app/policies/department_policy.rb

```diff
@@ -40,6 +63,14 @@ def create?
     @user.superadmin?
   end
 
+  def associate_user?
```

Comment:
As you just check for `@user.superadmin?` in the method, why not just use `@user.superadmin?`